### ISSUE TITLE
ClinVar Variant file doesn't contain the assembly field required in coordinates-only API submissions

### ISSUE THREAD

**Issue by northwestwitch** [Clinical-Genomics/scout#3693]
Using the ClinVar API, for SNVs without HGVS (and SVs probably) you must describe a `chromosomeCoordinates` field. This field is described by one of these 2:

```
"anyOf": [
            {
              "required": [
                "assembly",
                "chromosome"
              ]
            },
            {
              "required": [
                "accession"
              ]
            }
          ]
```

Accession (example: rs912919201) is easy but not always available. If you want to use the coordinates then you should also provide an assembly. The problem is that ClinVar submission spreadsheets don't contain assembly, since HGVS is mandatory for SNVs. 

<img width="316" alt="image" src="https://user-images.githubusercontent.com/28093618/205258296-4bd53903-a376-4f6c-ab7c-36857d48f37c.png">

We should solve the problem here in Scout by requiring that the user chooses an HGVS, but what about SVs?

Perhaps start by NOT including SVs in the submissions for now..

**Comment by dnil** [comment]
Would a possible workaround be to use the genome contig name plus g. coord for the HGVS, like NC_000001.11 g.100011_200022del? I haven't tried submitting SVs in a little bit, but that would have worked out pre API...

**Comment by northwestwitch** [comment]
Mmm I don't think so, by looking at the description of the [spreadsheet](https://www.ncbi.nlm.nih.gov/clinvar/docs/submit/#spreadsheet) and the [examples](https://ftp.ncbi.nlm.nih.gov/pub/clinvar/submission_examples/validation_examples/) of validated and not validated files. Specifically look at the valid_chr_coordonates ones.

BUT, leaving all as it is in Scout, when sending the request to PreClinVar to convert the files into json objects, we could provide the assembly as an additional request parameter `?assembly=GRCh37`. And let it add to the submission json.

Do you remember of providing the genome assembly was one of the several steps of using the ClinVar portal to submit using the submission files? I think so right?

**Comment by northwestwitch** [comment]
This will be soon fixed in preClinVar (https://github.com/Clinical-Genomics/preClinVar/pull/68)

**Comment by northwestwitch** [comment]
This was fixed in #3707. Closing!
